Adding a page to the Guides about Courses (#182)
arfc/arfc.github.io: manual/guides/courses.md

---
layout: manual
title: Course Planning & Registration
subtitle: "Assistance with Choosing and Registering for Classes"
permalink: /manual/guides/courses
---

# Undergraduate Students

A list of courses for all students can be found
[here](http://catalog.illinois.edu/courses-of-instruction/npre/).

> For undergraduate students, the NPRE department has compiled an interactive suggested course sequence for undergraduate students, which can be found

Reviewer: ```suggestion
For undergraduate students, the NPRE department has compiled an interactive 
suggested course sequence, which can be found
```

[here](https://npre.illinois.edu/academics/undergraduate/curriculum).
Undergraduate Students should meet with their faculty advisor each semester to

> discuss what other courses they need to take to satisfy their degree

Reviewer: ```suggestion
discuss career goals and what courses they need to take to satisfy their degree
```

> requirements. They should bring with them a copy of their DARS report

Reviewer: ```suggestion
requirements. Students should bring with them a copy of their DARS report
```

as well as a plan for the coming semester.

It's recommended for undergraduate students to stick to the suggested sequence
because some classes are offered only in one half of the school year.
The suggested sequence accounts for this and plans accordingly. For students
who test out of any classes, it's important that you talk with your
advisor and make sure there are no foreseeable conflicts.

# Undergraduate Checklist

```markdown

> - [ ] Looked at suggested course sequence

Reviewer: ```suggestion
- [ ] Review the detailed undergraduate checklist at 
      [npre.illinois.edu](https://npre.illinois.edu/academics/undergraduate/advising
- [ ] Review 
      [suggested course sequence](https://npre.illinois.edu/academics/undergraduate/course-sequence)
```

- [ ] Plan your classes for the following semester(s)

> - [ ] Talk with your faculty advisor

Reviewer: ```suggestion
- [ ] Set up an advising appointment with your faculty advisor at [my.npre](https://my.npre.illinois.edu/)
- [ ] Run a DARS audit at [http://registrar.illinois.edu/dars-audit](http://registrar.illinois.edu/dars-audit)
- [ ] Download and fill out a [course registration approval form](https://ws.engr.illinois.edu/sitemanager/getfile.asp?id=88)
- [ ] Review your DARS audit and course registration form with your faculty advisor
```

- [ ] Make any edits due to course load, class timing, etc.

> - [ ] Talk with Becky to release your registration hold

Reviewer: ```suggestion
- [ ] Email your course registration form (with advisor signature) to Becky 
      requesting a time to chat with her.
- [ ] Chat with Becky to release your registration hold.
```

> - [ ] Check your time ticket for registration

Reviewer: ```suggestion
- [ ] Check your time ticket for registration through Self-Service at 
      [Student Self Service](https://apps.uillinois.edu/selfservice).
```

> - [ ] Register during your registration period through Self Service

Reviewer: ```suggestion
- [ ] Register during your registration period at 
      [Student Self Service](https://apps.uillinois.edu/selfservice)
```

```

# Graduate Students

Each graduate student in ARFC should have a folder in Google Drive shared
between them and Prof. Huff. In that folder should be a Student Plan in the
form of a spreadsheet.

<img src="https://user-images.githubusercontent.com/393899/84724236-44cf4b80-af4d-11ea-9ac4-8ce729b72679.png" alt="example-spreadsheet" width="750"/>

Before registration for the coming semester, which is around the end of April
for the coming Fall semester or the beginning of November for the
coming Spring semester, graduate students should fill out their course plan
and discuss it with Prof. Huff. Use these links to find the course
requirements for the
[MS degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-ms/) and
[PhD degree](http://catalog.illinois.edu/graduate/engineering/nuclear-plasma-radiological-engineering-phd/).

Early Research Assistantship students should be taking 2-3 graduate classes
per semester and 4-8 research credit hours. Later, students typically
take one or fewer graduate classes per semester and fill the rest of their

> necessary hours with research credits.

Reviewer: ```suggestion
necessary hours (12) with research credits.
```

After Prof. Huff approves your course plan in the Student Record, you can
request the registration hold be lifted so that you can go ahead and
register for classes. To do so, send a polite email to the student office in
NPRE (specifically, the graduate student advisor staff member in NPRE),
CC'ing Prof. Huff, with a PDF copy of your student record document attached.
Let the staff member know that we have discussed the attached course
plan, Prof. Huff has approved it, and you would like to request the
registration hold be lifted.

Near the beginning of the semester, all students should update their Student
Record with the grades they have earned in the courses from the past
semester.

# Graduate Checklist

```markdown
- [ ] Fill out your Student Plan

> - [ ] Discuss your selection with Prof. Huff

Reviewer: ```suggestion
- [ ] Discuss your selection with Prof. Huff during a regular individual meeting.
```

- [ ] Make any edits due to course load, class timing, etc.

> - [ ] Request your registration hold lifted

Reviewer: ```suggestion
- [ ] Email plan to NPRE staff to request that your registration hold be 
      lifted. CC Prof. Huff.
```

> - [ ] Register for classes through Self Service

Reviewer: ```suggestion
- [ ] Register for classes through [Student Self Service](https://apps.uillinois.edu/selfservice).
```

- [ ] Register for Prof. Huff's Thesis Research using CRN 24115

> - [ ] Make sure you meet the credit requirements

Reviewer: Please add a link here.

Author: Do you just want a link that talks about the 12 credit-hour requirement?

> - [ ] Ensure your previous grades are up to date

Reviewer: ```suggestion
- [ ] Ensure your previous grades are up to date in the Student Record spreadsheet
```

```

# How to Register

Registration is done through
[Self Service](https://apps.uillinois.edu/selfservice/). First, click on the
link titled "*Registration & Records*". Most registration is through
"*Enhanced Registration*". From there, you can search and register for courses
and sections.

To register for Prof. Huff's Thesis Research, go instead to "*Classic
Registration*". Add the class "NPRE 599" and make sure to use the CRN number
24115 for Prof. Huff's section. The CRN numbers for all NPRE instructors can
be found [here](https://npre.illinois.edu/academics/courses/NPRE599-120198).
From there, you can change the amount of credits as necessary.

> # Useful Courses

Reviewer: I'm not sure this section needs to exist.

Author: Do you mean the section on how to register for courses? I just put that in for new grad students just in case, but you're right that it isn't entirely necessary.

Reviewer: Keep the registering for courses part. 
I'm referring to the Useful Courses section. 
This comment is attached to the line:
```
# Useful Courses
```

The logic is just that course recommendations really vary depending on the student.
NPRE247 and NPRE 455 are great, but they're required for undergrads, and I don't always recommend them for all grads... so it feels a bit strange to include them. I do recommend NPRE 555 and 560,  but I don't often teach them...

> - [NPRE 247: Modeling Nuclear Energy System](https://npre.illinois.edu/academics/courses/NPRE247-120191)

Reviewer: ```suggestion
- [NPRE 247: Modeling Nuclear Energy System](https://courses.illinois.edu/schedule/terms/NPRE/247)
```

> - [NPRE 455: Neutron Diffusion & Transport](https://npre.illinois.edu/academics/courses/NPRE455-120191)

Reviewer: ```suggestion
- [NPRE 455: Neutron Diffusion & Transport](https://courses.illinois.edu/schedule/terms/NPRE/455)
```

> - [NPRE 555: Reactor Theory I](https://courses.illinois.edu/schedule/2020/fall/NPRE/555)

Reviewer: ```suggestion
- [NPRE 555: Reactor Theory I](https://courses.illinois.edu/schedule/terms/NPRE/555)
```

Early Research Assistantship students should be taking 2-3 graduate classes per semester and 4-8 research credit hours. Later, students typically take one or fewer graduate classes per semester and fill the rest of their necessary hours with research credits.

After Prof. Huff approved your course plan in the Student Record, you can request that the registration hold be lifted so that you can go ahead and register for classes. To do so, send a polite email to the student office in NPRE (specifically, the graduate student advisor staff member in NPRE), CC'ing Prof. Huff, with a pdf copy of your student record document attached. Let the staff member know that we have discussed the attached course plan, Prof. Huff has approved it, and you would like to request the registration hold to be lifted.

> Once this is done, you can go through [Self Service](https://apps.uillinois.edu/selfservice/) and Register for Classes by going to "Registration & Records", then "Enhanced Registration".

Reviewer 2: Registration for Katy's section of Thesis research can only be done through "Classic Registration."

There are some additional steps required for adjusting the credit hours.

1. Enter the CRN for Prof. Huff's section
2. Update selection
3. Update credit hours
etc

These are the kind of specifics that would be helpful. e.g. I forget Katy's CRN every semester. I think its like 24115 but I'm not sure.

Author: How do you update the number of hours for the section? And what sort of certificates do you think are useful?

---
layout: manual
title: Course Planning & Registration

> subtitle: "Assistance with Choosing and Registering for Classes"

Reviewer: ```suggestion
subtitle: "Choosing and Registering for Classes"
```

Reviewer: Hi Gavin -- it looks like you've resolved this without committing the suggestion.... Let's discuss how to handle these change suggestions.
